platform/ui-functional-tests/src/test/java/org/motechproject/testing/uifunctionaltests/DataServicesUIFT.java

```diff
@@ -0,0 +1,35 @@
+package org.motechproject.testing.uifunctionaltests;
+
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.motechproject.uitest.TestBase;
+import org.motechproject.uitest.page.DataServicesPage;
+
+public class DataServicesUIFT extends TestBase {
+
+    public static final String ENTITY_NAME = "newEntity123456";
+
+    DataServicesPage dataServicesPage;
+
+    @Before
+    public void initialize() {
+        dataServicesPage = new DataServicesPage(getDriver());
+        login();
+    }
+
+    @After
+    public void cleanUp() throws InterruptedException {
+        logout();
+    }
+
+    @Test
+    public void newEntityTest () throws Exception {
+        dataServicesPage.goToPage();
```

Comment:
we should also assert a few things, for instance, did we land on the edit schema page with the newly created entity selected, is this entity now present in entity for schema edit select, is it present in the databrowser,